### PR TITLE
docs(core): #1150 add issue templates for feature, bug, and docs

### LINKED ISSUE #1150: Issue template for bug / feature etc
https://github.com/angular/material2/tree/master/.github/ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by richardhooper**
closes #1150

### Notes

Added 3 new templates to `.github/ISSUE_TEMPLATE/`, based more on GitHub defaults than the Material ones in the end (obviously further input is welcomed!).  Used GitHub settings on my forked repo, which adds commits directly to `origin`, resulting in some fun rebasing to get the changes into an issue branch to do a PR from...

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
```
Comment (eddiejaoude) [inline .github/ISSUE_TEMPLATE/bug_report.md:7]: Is this extra blank line needed? Not important, just my OCD 😂

**Comment by eddiejaoude** [comment]
Looks great 👍 , thank you 🤓

**Comment by eddiejaoude** [comment]
I do have 1 question, should this template be removed as part of this PR? `.github/ISSUE_TEMPLATE.md`

**Comment by eddiejaoude** [comment]
I think future improvements can come in another PR 🤓 . I dont see anything stopping us from accepting and merging this now 👍

**Review by eddiejaoude — APPROVED** [review]

**Comment by richardhooper** [comment]
> I do have 1 question, should this template be removed as part of this PR? `.github/ISSUE_TEMPLATE.md`

Yes, probably!  GitHub seems to ignore it now that we have the specific issue type templates in their own folder.

**Comment by eddiejaoude** [comment]
Are you happy to create a new PR and remove that file?

**Pull request merged**